src/flake8_boolean_trap/plugins.rs

```diff
@@ -5,6 +5,8 @@ use crate::ast::types::Range;
 use crate::check_ast::Checker;
 use crate::checks::{Check, CheckKind};
 
+const FUNC_NAME_WHITELIST: &[&str] = &["get", "setdefault", "pop", "fromkeys"];
```

Comment:
Consider the more inclusive term “allowlist”.

If we’re going with this method name heuristic, some other candidates that come to mind are `set.add`, `list.append`, `set.discard`, `list.count`, `list.index`, `list.insert`, `list.remove`. There could be a lot of these, even before considering methods of custom classes. I think we should maybe start with upstream’s approach of ignoring method calls entirely?

Reply:
Yeah, at least it's an independent check code so in theory can be opted out?

Reply:
It’s not independent from the code used for non-method function calls.

```console
$ cat test.py
def f(x):
    pass


f(True)
[].append(True)

$ flake8 test.py
test.py:5:1: FBT003 do not use boolean positional args. Hint: in `f(..)`, refactor positional arg #0 to include its argument name

$ ruff --select=FBT test.py
Found 2 error(s).
test.py:5:3: FBT003 Boolean positional value in function call
test.py:6:11: FBT003 Boolean positional value in function call

Reply:
Oh sorry, I misunderstood here. What do you think, @pwoolvett?

Reply:
My gut tells me many people will have fbt003 disabled. For it to be useful, we should have an explicit allowlist. Its not a pain to maintain, and prs are easy to merge: add test case with false positive and name to allowlist. list will grow and after a couple of iterations it should be stable. Dev will have to pay the price of potential false negatives, but its still better than no fbt003.

* Custom classes should be addressed by the dev when a bt is raised on codebase.
* For cases such as when a library emits a bt, eventually an option in the toml could be used to extend the allow list. But do note the main issue here is the signature for those stdlib methods explicitly disallows named arguments with the `/` token. Other cases are expected to be improved by the dev when possible by using named args.
* I have nothing against splitting func (fbt003)/methods (fbt004?), keeping comments above in mind.

note: whatever is decided here will be implemented upstream eventually to keep them in sync.

If you want to debate further, maybe discussions would be a better place, but i think theyre not ensbled...